tests/assertions/internal-libraries.n

```diff
@@ -80,18 +75,18 @@ let pub main = [_: ()] -> cmd[()] {
 
 	import websocket
 
-	assert type websocket.connect : websocket.connectOptions -> str -> cmd[maybe[str]]
+	assert type websocket.connect : { onOpen: [websocket.send] -> cmd[bool]; onMessage: [websocket.send] -> cmd[bool] } -> str -> cmd[maybe[str]]
```

Comment:
Ah, oops, I meant to export `websocket.connectOptions` as an alias (like `websocket.send`), but if that never happened, then I guess `websocket` doesn't export it?

https://github.com/nbuilding/N-lang/blob/b95c92dc027cdc43d56b1e285bc8cc111728f0d8/python/libraries/websocket.py#L16-L25

Reply:
Yeah it was not exported